tasks/internal/push-artifacts-to-cdn-task/push-artifacts-to-cdn-task.yaml

```diff
@@ -123,11 +123,12 @@ spec:
                   > "$(results.result.path)"
                 if [ -f "$STDERR_FILE" ] ; then
                     tail -n 20 "$STDERR_FILE" >> "$(results.result.path)"
+                    cat "$(results.result.path)"
                 fi
             fi
-            exit 0 # exit the script cleanly as there is no point in proceeding past an error or exit call
+            exit "$err"
```

Comment:
Why this change? AFAIR, the idea was to be able to view the error in the managed pipeline that called it, the IR pipeline needs to succeed.

Reply:
yes. I reverted this and re-worked it so that internal-pipelineruns can be used too